client/src/content/content.js

```diff
@@ -0,0 +1,27 @@
+import registerContentProxy from "@cd/services/ServiceWorker/Provider";
+
+function injectCustomJs() {
+    try {
+      let jsPath = '/static/js/scripts/content/inpage.js';
+      const container = document.head || document.documentElement;
+      const scriptTag = document.createElement('script');
+      scriptTag.setAttribute('type', 'text/javascript');
+      scriptTag.src = chrome.runtime.getURL(jsPath);
+      container.insertBefore(scriptTag, container.children[0]);
+      scriptTag.onload = function () {
+        // remove after run the script
```

Comment:
Can you please add more comment here to explain why we need to remove the script after executing it? Thanks

Reply:
Hello @duc-q bro, please take a look, I commented on it